src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -17,7 +17,8 @@ public static class BitConverter
         // The value is set to true if the architecture is
         // little endian; false if it is big endian.
         [Intrinsic]
-        // It's an intrinsic in all runtimes and doesn't need a static constructor
+        // It's an intrinsic in all runtimes and so doesn't need to be initialized here.
+        // Not adding such initialization helps avoid BitConverter getting a static cctor.
         public static readonly bool IsLittleEndian;
```

Comment:
nit: Can we add a test that validates reflection works as expected?

Reply:
Added